vcpkg/TOC.yml

```diff
@@ -3,432 +3,434 @@
 - name: Get started
   expanded: true
   items:
-  - name: vcpkg overview

```

Comment:
In general there are a lot of spacing changes in the TOC file in this PR, is it possible to revert those so we can just see the added Dependabot article?

Reply:
Done